### PR TITLE
Task#68068: BE Xem danh sach lich hen

### PR DESCRIPTION
**Pull request by dangvm-0369**
## Related Tickets:
- [Task#68068](https://edu-redmine.sun-asterisk.vn/issues/68068)

## WHAT this PR does?
- Backend xem danh sách lịch hẹn

## Notes Impacted Areas:
-Model, Controller, routes
## Attachments:
- N/A.

### CODE REVIEW

**Review by tungbd-0214 — APPROVED** [review]

**Comment by dangvm-0369** [comment]
ready

File: app/Http/Controllers/OrderController.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return Inertia::render('orders/Index.jsx', [
+           'orders' => Order::with(['products', 'customer'])>limit(10)->get(),
```
Comment (ghost) [inline app/Http/Controllers/OrderController.php]: 10 bỏ vào config

**Comment by dangvm-0369** [comment]
ready

**Comment by dangvm-0369** [comment]
ready

**Pull request merged**